parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -844,6 +844,22 @@ ensureGPUDrivers() {
     fi
 }
 
+# TODO: this is a temporary ubuntu-only HACK until we get a driver
+ensureAMDGPUDrivers() {
+    echo "Installing AMD GPU drivers"
+
+     # delete amdgpu module from blacklist
+    sudo sed -i '/blacklist amdgpu/d' /etc/modprobe.d/blacklist-radeon-instinct.conf
+
+    pushd /var/cache/amdgpu-apt
+    ls -l
+    sudo dpkg -i *.deb
+    popd
+
+    REBOOTREQUIRED=true
```

Comment:
this is causing us customer headache, we need to ensure the node doesn't get to a ready state before the reboot.

Reply:
I though setting the variable automaticall does it for me. Any guidance on what needs to change?

Reply:
@ganeshkumarashok what are we planning to do for the MIG use case ?